src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -13,6 +13,6 @@ internal static class FeatureFlags
         /// Enable String Interpolation feature. 
         /// Added 12/3/2021.
         /// </summary>
-        public static bool StringInterpolation = false;
+        public static bool StringInterpolation = true;
```

Comment:
Consider taking this flag toggle as a separate change, after updating on the Canvas side.

Reply:
Removed the feature flag change